docs/cookbook/README.md

```diff
@@ -0,0 +1,7 @@
+# The Gradle Cookbook
+
+This is a collection of guides and examples for the Gradle Build Tool.
+
+## Table of contents
+* [Gradle Build Tool Architecture](/community/gradle-architecture/architecture/)
```

Comment:
Not sure it will be a part of the cookbook (a set of recipes for end users). It seems to be rather a part of the contributor / developer section

Reply:
Fixed.